docs/avro.rst

```diff
@@ -145,22 +98,94 @@ simply dropped.  Had the new schema dropped the original field ``favorite_number
 forward compatible with the original user schema since we wouldn't know how to fill in the value for ``favorite_number``
 for the new data because the original schema did not specify a default value for that field.
 
+If the compatibility is set to ``FORWARD_TRANSITIVE`` (not just ``FORWARD``), then it means that consumers using all previousely registered schemas (not just the latest schema) can read data written by producers using the new schema.
```

Comment:
How about this: FORWARD_TRANSITIVE compatible schema means that all versions (X, X-1, ..., 1) of the schema can be used to decode data encoded with version X of the schema.

Reply:
Good idea. I've adapted this suggestion